### PR TITLE
Add Delete Client/Property section in User Guide

### LINKED ISSUE #148: [PE-D][Tester D] Delete Property has no info on how to execute command in UG
-------------
Labels: `type.DocumentationBug` `severity.Low`
original: abhityakrishnaraj/ped#1

### LINKED ISSUE #153: [PE-D][Tester E] Format for delete-client command is not stated in UG
-------------
Labels: `type.DocumentationBug` `severity.Medium`
original: jorellesee/ped#6

### PR DESCRIPTION
**Pull request by FeliciaBeatrice**
Fixes #153 
Fixes #148

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -104,7 +104,11 @@ Lastly, for valid `PRICE`, a positive number excluding any letters/symbols/space
 ---
 
 ### Delete Property: `delete -property`
+Deletes the specified property from the property list and subsequently deletes any pairings involving that property.
```
Comment (wilsonngja) [inline docs/UserGuide.md:107]: I really like how you keep your explanation concise.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -153,6 +157,11 @@ For valid `BUDGET_MONTH`, a positive number excluding any letters/symbols/spaces
 ---
 
 ### Delete Client: `delete -client`
+Deletes the specified client from the client list and subsequently deletes any pairings involving that client.
+
+<u>Format:</u> `delete -client ic/CLIENT_INDEX`
+
+<u>Example:</u> `delete -client ic/3`
```
Comment (wilsonngja) [inline docs/UserGuide.md:164]: I really like how clear a explanation is.
Maybe can also consider having an expected result?

**Review by wilsonngja — COMMENTED** [review]
Good job in the improvement to the UG. LGTM

**Review by ngdeqi — APPROVED** [review]
Could use a bit more info, but LGTM

**Pull request merged**